### ISSUE TITLE
Issues running PyAF

### ISSUE THREAD

**Issue by anuliketat** [antoinecarme/pyaf#110]
Hi @antoinecarme 
1. Like I mentioned earlier I have the same problem as the one on this thread. I don't know if the kernel hangs or it is taking too much time to train the data. My data is just a pandas dataframe of two columns with a date time column of 365 days and respective sales column for those dates. I have arranged everything in the exact format as you mentioned in the PyAF documentation and sample code. 
I have attached the snapshot..

![image](https://user-images.githubusercontent.com/11734945/51570514-2fbfcb80-1ec5-11e9-8391-ecd99633ea9c.png)

2. After installing PyAF, I got this error while trying to load some packages - *'tuple object has no attribute type'*. 
I got it resolved after finding that numpy, pandas and sklearn packages got installed again with different versions during the installation of pyAF and other mentioned packages. Also other than those three many other packages didn't load throwing the same error. I don't know which one is doing this.

3. Is there a clear description of what auto forecast engine is doing in the background or is it a black box? How the training is done and what's the metric it takes for validation. Also it would be great if you could provide an article on how auto forecast is doing differently or if better than other popular forecasting techniques like ARIMA, prophet etc.
Thank you.

_Originally posted by @anuliketat in https://github.com/antoinecarme/pyaf/issues/39#issuecomment-456598548_

**Comment by antoinecarme** [comment]
@anuliketat 

I will start with point 3. see

https://github.com/antoinecarme/pyaf/issues/35

There is a link to  a notebook in a comment with some description of algorithmic aspects of PyAF.

PyAF is a tool for automating some known methods for forcasting. It is a yet-another-foerecasting method. 

I do not yet have a benchmark against existing tools and my goal is not to produce the best  one but an automated method to produce reasonable forecasts (quality = MAPE, by default, can be changed) i na resonable time (performance/speed).

You can play with it and give your feedback.

**Comment by antoinecarme** [comment]
For the first point, do you ahve a proper python install ? when creating this issue, you were asked to provide some inf o : 

3. information on different software versions used (pyaf, numpy, pandas, scikit-learn etc). The output of the following script should be enough : https://github.com/antoinecarme/pyaf/blob/master/tests/basic_checks/platform_info.py

**Comment by anuliketat** [comment]
> For the first point, do you have a proper python install ? when creating this issue, you were asked to provide some inf o :
> 
> 1. information on different software versions used (pyaf, numpy, pandas, scikit-learn etc). The output of the following script should be enough : https://github.com/antoinecarme/pyaf/blob/master/tests/basic_checks/platform_info.py

Does it help?

![image](https://user-images.githubusercontent.com/11734945/51573011-90eb9d00-1ecd-11e9-8100-b8adb1a506ad.png)

**Comment by antoinecarme** [comment]
YES. Thanks. PyAF was not tested with python versions above 3.7. You are using 3.6.5, which is OK.
PyAF is tested after each commit , travis-ci.

https://travis-ci.org/antoinecarme/pyaf

A lot of models are built and compared with stored golden files. The process was last run 42 minutes ago.
 
I cannot really help with your python install. You need to see with a local expert.

Can you please share your dataset and a python notebook ? (ananymization is welcome).

**Comment by antoinecarme** [comment]
Closing issue after no response for 30 days. Not blocking.

Please repoen if needed.
